### PR TITLE
Check if useRef variable is not null

### PR DESCRIPTION
**Pull request by aaschlote**
Trying to fix the issue of the image below

image

### CODE REVIEW

File: app/components/cookieBanner/__test__/CookieBanner.test.tsx [hunk 1]
```diff
@@ -17,13 +17,13 @@ describe("CookieBanner", () => {
   };
 
   it("renders content without link when no link props given", () => {
-    const router = createMemoryRouter([
+    const CookieBannerWithRemixStub = createRemixStub([
```
Comment (pgurusinga) [inline app/components/cookieBanner/__test__/CookieBanner.test.tsx:20]: We can use the `render()` function, similar to the last test I mentioned.

File: package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
     "react-dom": "^18.3.1",
     "react-dropzone-esm": "^15.0.1",
     "react-imask": "^7.6.1",
+    "react-router-dom": "^6.26.1",
```
Comment (pgurusinga) [inline package.json:78]: can you elaborate on why we added `react-router-dom` and the only place we use directly `react-router-dom` you removed it
image

Comment (aaschlote) [reply package.json:78]: I was checking some threads in the stackoverflow and I read that the problem might be the usage of the `react-dom` library, so I removed it from the tests. Unfortunately it didn't work, so I decided to install it (someone mentioned, to usage the latest version), and again it didn't work.

**Comment by aaschlote** [comment]
Closing this PR for now, I'll check in another time

**Pull request closed without merging**